src/router/events.ts

```
@@ -0,0 +1,29 @@
+import { db } from "@/lib/db/db";
+import { os } from "@orpc/server";
+
+export const listEvents = os.handler(async ({ }) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)